content/en/blog/_posts/2021-05-30-update-with-ingress-nginx.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog
+title: 'Updates to Ingress Nginx with network v1'
+date: 2021-05-30
+slug: update-with-ingress-nginx
+---
+
+**Authors:** James Strong, Richardo Katz 
+
+With all Kubernetes APIs, there is a process to creating, maintaining, and
+ultimately deprecating them once they become GA. The network API is no
+different. With the release of Kubernetes 1.22, the network API goes from
+v1beta1 to v1, with v1beta is deprecated. This move has been in discussion
```

Comment:
Did you perhaps mean:
```markdown
different. With the release of Kubernetes 1.22, SomeResourceName graduates to stable.
```
?

Ingress, IngressClass and NetworkPolicy are all already GA so I'm not sure what's graduating in v1.22.

Reply:
Is the entire v1beta deprecated or just v1beta1/Ingress

Reply:
See https://kubernetes.io/docs/reference/using-api/deprecation-guide/ for details.

> The networking.k8s.io/v1beta1 API version of IngressClass will no longer be served in v1.22.
> The extensions/v1beta1 and networking.k8s.io/v1beta1 API versions of Ingress will no longer be served in v1.22. 

I think those are the relevant deprecations for the ingress-nginx code.

Reply:
updated using this wording.